modular_skyrat/modules/mapping/code/lavaland_ruin_code.dm

```diff
@@ -81,6 +92,20 @@
 	uniform = /obj/item/clothing/under/syndicate/skyrat/tactical
 	suit = /obj/item/clothing/suit/hooded/wintercoat/syndicate
 
+/datum/outfit/job/Lavaland_Vet
```

Comment:
also as i said in discord does this really need to be a `job/` subtype

Reply:
weird... that's .. not what it is currently